src/components/Admin/Filters.jsx

```diff
@@ -36,7 +36,8 @@ const Filters = ({ filters, setFilters, setObjects, objects, input }) => {
             } cursor-pointer flex items-center w-fit m-0`}
           >
             <p className="my-0 mx-1 px-2 py-[2px] whitespace-nowrap">
-              {filter}
+              {filter +
+                (filter === "accept" || filter === "reject" ? "ed" : "")}
```

Comment:
we will set up a standard across for all pages, so we dont have to do any conditional stuff to account for different pages